scripts/get.url.js

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const BaseScript = require('../lib/abstract-script')
+const utils = require('../lib/utils')
+
+class GetUrl extends BaseScript {
+  async run (options = {}) {
+    const taskName = 'GetUrl'
+    this.emit('start', taskName)
+    const urls = {}
+    if (options.action) {
+      const action = this.config.manifest.package.actions[options.action]
+      if (!action) {
+        throw new Error(`No action with name ${options.action} found`)
+      }
+      this.config.manifest.package.actions = {}
+      this.config.manifest.package.actions[options.action] = action
+    }
+    const actionUrls = await utils.getActionUrls(this.config, true)
```

Comment:
shouldn't we also return the ui url? or as part of a later PR

Reply:
For now, get.url.js is doing this ie. return both URLs. We can keep the utils function unchanged to return only URLs relevant as per the params passed to it.

Reply:
No that's fine, returning the two urls should be ok, but I would additionally return the UI/webassets url

Reply:
[ closing the loop ] UI urls to be added to a later PR